src/python/turicreate/toolkits/sound_classifier/sound_classifier.py

```diff
@@ -19,6 +19,74 @@
 from turicreate.toolkits._model import CustomModel as _CustomModel
 from turicreate.toolkits._model import PythonProxy as _PythonProxy
 
+USE_TF = _tk_utils._read_env_var_cpp('TURI_SC_USE_TF_PATH')
+
+class _Accuracy(object):
+    '''
+    Defines a common interface around MXNet/TensorFlow accuracy metrics.
+    '''
+
+    def __init__(self):
+        raise NotImplementedError
+
+    '''
+    Tallies the results from a single batch of predictions.
+
+    The predictions are expected to contain (possibly unnormalized) class
+    probabilities. Replacing the last axis of the predictions with the argmax
+    should yield a shape matching the ground truth.
+    '''
+    def update(self, ground_truth, predicted):
+        raise NotImplementedError
+
+    '''
+    Removes all tallied results so far.
+    '''
+    def reset(self):
+        raise NotImplementedError
+
+    '''
+    Computes the accuracy for all the results tallied so far.
+    '''
+    def get(self):
+        raise NotImplementedError
+
+class _MXNetAccuracy(_Accuracy):
+    def __init__(self):
+        import mxnet as mx
+        self.impl = mx.metric.Accuracy()
+
+    def update(self, ground_truth, predicted):
+        import mxnet as mx
+        self.impl.update(mx.nd.array(ground_truth), mx.nd.array(predicted))
+
+    def reset(self):
+        self.impl.reset()
+
+    def get(self):
+        _, result = self.impl.get()
+        return result
+
+class _TFAccuracy(_Accuracy):
+    def __init__(self):
+        import tensorflow.compat.v1 as tf
```

Comment:
Can you use`tensorflow.keras.metrics.Accuracy` not `tensorflow.compat.v1.keras.metrics.Accuracy`?

Reply:
Ah, good point. That change appears to work fine!